tutorials/katacoda/README.md

```diff
@@ -2,8 +2,17 @@
 
 Definitions of our courses for Katacoda.
 
-## Development.
+## Development
 
-Find docs [here](https://katacoda.com/docs)
+Process of adding / editing tutorials:
 
-Thanos repo is hooked to [this](https://katacoda.com/bwplotka) Katacoda account.
\ No newline at end of file
+1. Signup to Katacoda, link your fork of Thanos to your account, see [this](https://katacoda.com/docs/configure-git)
+1. Add/edit tutorials
+1. Push to `master` of your fork
+   1. You should see the updated preview of the tutorials on https://katacoda.com/<your-profile>
+1. Make a PR to Thanos repo.
```

Comment:
Looks like half the steps end in a `.` and the others don’t. Let’s make it consistent.